bigc/resources/products.py

```diff
@@ -41,3 +41,22 @@ def get(self, product_id: int, *, include_variants: bool = False, include_custom
         url_parts = url_parts._replace(query=urlencode(query_dict))
 
         return self._v3_client.request('GET', urlunparse(url_parts))
+
+    def create(self, *, name: str, product_type: str, weight: float, price: float, **kwargs) -> dict:
+        """Create a product"""
+        payload = {
+            'name': name,
+            'type': product_type,
+            'weight': weight,
+            'price': price,
+            **kwargs,
+        }
+        return self._v3_client.request('POST', '/catalog/products', json=payload)
+
+    def update(self, product_id: int, **kwargs) -> dict:
```

Comment:
https://github.com/MedShift/bigc/blob/master/bigc/resources/webhooks.py#L28

using a `data` kwarg seems like the established way to pass data into update requests so far.

Reply:
To be honest, we haven't been particularly consistent.